mne/preprocessing/interpolate.py

```diff
@@ -0,0 +1,64 @@
+"""Tools for interpolation data."""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+
+from itertools import chain
+import numpy as np
+
+from ..epochs import BaseEpochs
+
+
+def interpolate_mark_bads(insts, good_fraction=0., copy=True):
```

Comment:
The default I think should be to always just mark all bad (this seemed to be the consensus when we discussed), so `good_fraction=1.`.